### ISSUE TITLE
RA crashes when you turn off threaded video

### ISSUE THREAD

**Issue by GunseikanMikieru** [cizia64/CrossMix-OS#149]
Threaded video seems to be on for all RA cores by default. It is my (limited) understanding that threaded video does more bad than good for below-PS1 systems. Tried turning it off in some cores: RA crashes out of the game and restarts with no rom loaded and threaded video still enabled. Could try manually edition the config file as a workaround I guess.

**Comment by cobaltgit** [comment]
Try setting `video_threaded` to false in the Retroarch cfg file for mame2003plus, think it's in `/SDCARD/RetroArch/.retroarch/config/MAME 2003 Plus` or something

**Comment by cizia64** [comment]
I don't think that this is still true since CrossMix v1.2.1: with alsa sound output it was true that disabling threaded video has a positive impact on some low demanding cores, but since we use OSS audio output the performance are better with threaded video with any emulator so it's probably a bad idea to disable it.
If you have bad performances I would recommend to use another arcade emulator for double dragon. However that's surprising that this one is not working perfectly on Mame2003 Plus.

Can you run OTA update first please ?

**Comment by GunseikanMikieru** [comment]
I ran OTA update, but I already was on version 1.2.1 apparently. I found this video where someone solves DD slowdown by overclocking the PCB CPU's: https://www.youtube.com/watch?v=G182_TlVl9A
But is it even possible to do this in the RA version op mame2003plus?

**Comment by 2Bane** [comment]
The menu is accessible but those advanced options aren't. (Quickmenu/Core Options/Display MAME menu)

Video looks more like MAME standalone on PC not any retroarch core.
